events.json

```diff
@@ -1,183 +0,0 @@
-[{
```

Comment:
yeah the json was originally never really meant to be part of the repo so moving it to a submoule would proabbly be a good idea